### ISSUE TITLE
Delete pre-existing codewind workspaces via Che API in test automation

### ISSUE THREAD

**Issue by lentheivendra** [eclipse/codewind#1234]
Currently the automated tests on Kube for pfe/turbine and the sidecar delete/cleanup pre-existing workspaces by querying for running codewind pods and discovering the workspace ID to delete.  This does not work for any workspaces that are stopped, or for the multiple workspaces case.  Proposing solution to query for all workspaces via Che API and then filter out the codewind ones to delete as part of test cleanup.

**Comment by ssh24** [comment]
@lentheivendra Does the SC test remove the workspace after the test is done? I noticed the functional tests fail to remove it after the tests https://github.com/eclipse/codewind/issues/1239

**Comment by lentheivendra** [comment]
@ssh24 Yes the SC tests do remove the workspace at the end and it's been working so far.  Note that they remove it by using the workspace ID captured from the creation API's return info, rather than discovering the ID by inspection of pods, etc.

**Comment by ssh24** [comment]
I am wondering if the same is possible to do it for the functional test scripts as well

**Comment by maysunfaisal** [comment]
@ssh24 we should also do like how Len is doing via PR https://github.com/eclipse/codewind-che-plugin/pull/112

**Comment by elsony** [comment]
Done the sidecar piece of work. Still need to do the same on the Codewind PFE turbine test.
